src/clj/rems/form_validation.clj

```diff
@@ -2,7 +2,12 @@
   "Pure functions for form validation logic"
   (:require [clojure.string :as str]
             [rems.common.form :as form]
-            [rems.common.util :refer [+email-regex+ +phone-number-regex+]]))
+            [rems.common.util :refer [+email-regex+
+                                      +phone-number-regex+
+                                      +valid-ip-address-regex+
+                                      +valid-ip-address-regex-version-six+
+                                      +reserved-ip-address-range-regex+
+                                      +reserved-ip-address-range-regex-version-six+]]))
```

Comment:
suggestion: since the phone number & ip address regexes are only used in this namespace, they could be here instead of in common.util. The email-regex is also used for fields formats on the frontend, that's why its in common.util.

Reply:
I think having them in utils makes sense, for now, having an email regex in one place and phone/ip regex in another can be confusing

Reply:
If you do not mind, Ill leave it the way it is